src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -255,7 +255,7 @@ static bool is_pinvoke_override_library (const char* libraryName)
 #endif
 
 void
-mono_ios_runtime_init (void)
+mono_ios_runtime_init (int argc, char** argv)
```

Comment:
I just bumped into this: if we pass the arguments here, do we still need to do it again on line: https://github.com/dotnet/runtime/pull/92772/files#diff-6cef647106df3b37e45599e150f97a7dc3678f2606723b1967dbda7d1f0a42b9R281 
?

Reply:
Good catch, let's simplify it. Since the `mono_ios_runtime_init` exits I suggest retrieving arguments within the function.